src/BuildScriptGenerator/Ruby/RubyPlatform.cs

```diff
@@ -114,6 +114,13 @@ public PlatformDetectorResult Detect(RepositoryContext context)
 
             var buildProperties = new Dictionary<string, string>();
 
+            string runBuildCommand = Environment.GetEnvironmentVariable("RUN_BUILD_COMMAND");
```

Comment:
The `RUN_BUILD_COMMAND` string is used in the Python PR (and I believe other PRs) -- maybe we should put this string in a global file that they can all use?